### DISCUSSION TITLE
can I upgrade to fedoraremix 39 if I downloaded it directly from github releases?

### DISCUSSION THREAD

**Discussion by seanmamasde** [WhitewaterFoundry/Fedora-Remix-for-WSL#208] [Q&A] [answered]
I don't want to lose all configs/settings from fedoraremix38, or will dnf system update work (sounds stupid)?

**Comment by crramirez** [comment, marked as answer]
Make a backup and
Follow this: Fedora Remix for WSL 38 Released — Whitewater Foundry
<https://www.whitewaterfoundry.com/blog/2023/05/03/fedora-remix-for-wsl-38-released>
just
change 38 by 39. Soon we'll publish a new post.

**Reply by seanmamasde** [reply]
thank you! will look into that post.
